tests/contrib/psycopg/test_psycopg_opentracer.py

```diff
@@ -0,0 +1,64 @@
+# 3p
+import psycopg2
+import opentracing
+
+# project
+import ddtrace
+from ddtrace.contrib.psycopg.patch import patch, unpatch
+
+# testing
+from tests.contrib.config import POSTGRES_CONFIG
+from tests.opentracer.utils import opentracer_init
+
+
+class TestPsycopgPatchOpentracer(object):
+    """Test using both the Datadog and opentracer tracers."""
+
+    def setUp(self):
+        self._dd_tracer = ddtrace.tracer
+        self.ot_tracer, self.dd_tracer = opentracer_init()
+        self.writer = self.dd_tracer.writer
+
+    def tearDown(self):
+        unpatch()
+
+        ddtrace.tracer = self._dd_tracer
+
+    def test_simple_trace(self):
```

Comment:
this test is redundant since it's a subset of the other one. You can remove it because we need to test only the integration.